src/biogeophys/TemperatureType.F90

```diff
@@ -1432,6 +1448,22 @@ subroutine UpdateAccVars (this, bounds)
 
     call update_accum_field  ('T10', this%t_ref2m_patch, nstep)
     call extract_accum_field ('T10', this%t_a10_patch, nstep)
+    
+    do p = begp,endp    
+       c = patch%column(p)  
+       rbufslp(p) = this%t_soisno_col(c,3)      
+    end do
```

Comment:
I'm creating a new issue #952, as the `stress_decid` phenology scheme also uses the 3rd soil layer as a trigger.  The solution for this issue should be consistent with what we end up doing here. https://github.com/ESCOMP/CTSM/blob/384c726fa74e5a16ae15f6b40aad3e5d63b19d07/src/biogeochem/CNPhenologyMod.F90#L1122

Reply:
@lmbirch89 we should set the depth you need this from in the Init subroutine in TermperatureMod by looking at zsoi. As @wwieder points out we'll solve this for #952, and apply the same solution here.

Reply:
Let's talk about the requirements for this depth on Wednesday. Is there a problem with the layer at the right depth being too thick? Or is just the depth important?

Reply:
With the changes I recently made on master (which have been merged into this branch), this can use the `phenology_soil_layer` parameter defined in CNPhenologyMod. Ideally, this code from TemperatureType would probably be moved into a CNPhenologyType in CNPhenologyMod, since we like to keep data local to the module that needs it. However, I don't feel that *needs* to be done. However, with the current organization, there would be a circular dependency between CNPhenologyMod and TemperatureType, so *something* needs to be moved.

Reply:
This part has been done.